lib/tasks/repush_all_handover_dates_to_delius.rake

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+desc 'Recalculate handover dates for all known offenders, and push changes into nDelius'
+task repush_all_handover_dates_to_delius: :environment do
+  Rails.logger.info 'Invoking rake task :recalculate_handover_dates'
```

Comment:
If you get no log output in Kibana you may have to add `Rails.logger =  Logger.new($stdout)` here